.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This file isn't necessary and should be added to .gitignore

Reply:
added to gitignore